.github/workflows/node.js.yml

```diff
@@ -23,6 +23,6 @@ jobs:
       - name: Deploy 🚀
         uses: JamesIves/github-pages-deploy-action@releases/v3
         with:
-          ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This is provided by GitHub.
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }} # Use a different Personal Access token personal access token you generated with at least public_repo, read:org, workflow access by adding it to the Settings/Secrets part of this repository with anme ACCESS_TOKEN
```

Comment:
@saadpasta See in the docs what they do.
https://github.com/marketplace/actions/deploy-to-github-pages#getting-started-airplane

Reply:
@naveen521kk we forgot to change the key